src/elements/stream_element.ts

```diff
@@ -97,7 +98,11 @@ export class StreamElement extends HTMLElement {
   }
 
   private get targetElementsById() {
-    const element = this.ownerDocument?.getElementById(this.target!)
+    if (!this.target) {
+      return [];
+    }
+
+    const element = this.ownerDocument?.getElementById(this.target)
```

Comment:
This seems like a regression in both readability and style. Is there a way to avoid this?

Reply:
we could allow non-null assertions if its likely they'll be common in the codebase.

IMO we should avoid using them so there's less chance of someone forgetting to do a null check.

another option is to disallow them but add an ignore comment to this particular line and the one below. there's only two so its probably better to make those the exception rather than turn the rule off